### ISSUE TITLE
Publishing on F-Droid

### ISSUE THREAD

**Issue by Jowat97** [graphefruit/Beanconqueror#601]
Hello @graphefruit,

first at all thank you very much for this app and your contribution. It is very nice to see an app like this as FOSS software. However, unfortunately it is necessary to have the Google Play Store and a Google account to install this app on Android devices.

Therefor I would really appreciate it, if this app could be published on F-Droid. A free and open source Android App Store with only FOSS apps. If you need any help to publish your app on F-Droid feel free to ask!

Would welcome your response.

**Comment by graphefruit** [comment]
Hello @Jowat97,
a while ago back I've watched to F-Droid, and when I understood it rightly, F-Droid doesn't support an own account where I can upload my APK's my self, and rather do a support request, to get the app build by their building system.
Therefore if errors occures, I cant grab into, fix it or whatever.

So actually I don't plan to support F-Droid.
You can download the APK directly here on Github, without having Google Play store or a google account.

If you can help to lower the pain to get on F-Droid I'm happy for help.

Best
Lars

**Comment by Jowat97** [comment]
Yes, it is the main feature of F-Droid, that all apps are build and signed by an trusted authority, to ensure that nothing got compromised or an malicious apk was published. The whole building process like logs etc. are publicly available and the generated apks are reproducible. And the building process shouldn't be a problem at all. Although the app will be build by F-Droid, you are the one defining how to build it. So if you can compile and build it, F-Droid will be able do to the exact same thing, see: https://f-droid.org/docs/Submitting_to_F-Droid_Quick_Start_Guide/

Though I think it is really sad not to have this app on F-Droid, which would be an enrichment and could also lead to more users for this app, I respect your decision. 

BR

**Comment by graphefruit** [comment]
Thanks for the insights.
If you want to give a helping hand how to write all this I'm happy to take.
Another user in Discord asked the same question and he mentioned izzydroid, wouldn't be this another possibility?

**Comment by Jowat97** [comment]
Unfortunately I am also not an expert in this case, but I could try my best.
And no, izzydroid is a person who has his own repository. It mostly contains app, which are not (yet) on F-Droid for several reasons. For example if they don't compliance with the guidelines, because they use proprietary libraries, have trackers included etc.
Apps from his repository are not included in F-Droid. You would have to add his repository to F-Droid manually and there wouldn't be any real benefits compared to just install your apk from GitHub.

**Comment by graphefruit** [comment]
I see, thanks for insights.
I don't know how F-Droid handles this, after I'm also having a analytics (matomo) build in, but which can be enabled/Disabled by user, defaults its off.
Would this be against the guidelines?

**Comment by Jowat97** [comment]
No, analytics aren't forbidden by default. Just some analytics libraries like google analytics (firebase).
The guidelines can be found here: https://f-droid.org/de/docs/Inclusion_Policy/
I try to open a ticket at F-Droid to publish your app. Your code will be scanned for forbidden libraries. So afterwards we would now for sure. But at a first glance it seems to be fine.

**Comment by Jowat97** [comment]
A submission request was created: https://gitlab.com/fdroid/rfp/-/issues/2545 :)

**Comment by graphefruit** [comment]
Thanks we'll see how its going.
Just bump here, I'll also try to have a look into the fdroid ticket.
Lars

**Comment by graphefruit** [comment]
After this topic is now open for 1 1/2 years, I gonna close this ticket now, as unplanned.
